sdk/core/Azure.Core.Experimental/src/DynamicDataOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.Dynamic
+{
+    /// <summary>
+    /// Provides the ability for the user to define custom behavior when accessing data, such as JSON, through a dynamic layer.
+    /// </summary>
+    public struct DynamicDataOptions
+    {
+        /// <summary>
+        /// Gets the default <see cref="DynamicDataOptions"/> for Azure services.
```

Comment:
Perhaps in a different PR, but could you clarify exactly what is considered "default" for Azure services? As a caller, I feel this is rather opaque and not likely to use it when I can be more specific with another overload.

Reply:
Sure! https://github.com/Azure/azure-sdk-for-net/issues/35281